src/Module/FollowConfirm.php

```diff
@@ -0,0 +1,93 @@
+<?php
+namespace Friendica\Module;
+
+use Friendica\App;
+use Friendica\BaseModule;
+use Friendica\Core\L10n;
+use Friendica\Core\Logger;
+use Friendica\Core\Protocol;
+use Friendica\Database\DBA;
+use Friendica\Model\Contact;
+use Friendica\Protocol\Diaspora;
+use Friendica\Protocol\ActivityPub;
+use Friendica\Util\DateTimeFormat;
+
+class FollowConfirm extends BaseModule
+{
+	public static function post()
+	{
+		$a = self::getApp();
+
+		$uid = local_user();
+		if (!$uid) {
+			notice(L10n::t('Permission denied.') . EOL);
```

Comment:
```suggestion
			throw new \Friendica\Network\HTTPException\ForbiddenException(L10n::t('Permission denied.'));
```

Reply:
Since it's in `post()` it would show a bare screen, which we probably don't want.